rfc-135-updating-routes-in-router.md

```diff
@@ -0,0 +1,44 @@
+# Updating routes in Router
+
+## Summary
+
+This RFC proposes changing the way that routes are updated in Router, moving away from Router API manually updating Router instances and instead allowing Router instances to keep themselves up-to-date via polling MongoDB for changes.
+
+## Problem
+
+Whenever a route needs to be added or updated, Content Store calls into Router API to perform the update. Router API knows the address of every Router instance from a static file that it reads at start-up; it then uses this information to call the `/reload` endpoint on every Route instance to let them know that routes have been updated, so that those instances reload the routes they hold in memory.
```

Comment:
I wonder if it's worth adding links to how this works to make it easier to find more information. I'm thinking for example https://github.com/alphagov/govuk-puppet/blob/84689ae2c36bbc7740da6a8d70c7f4e04d0e2357/modules/govuk/manifests/apps/router_api.pp#L94-L118 or https://github.com/alphagov/router-api/blob/2aa74d402624837c87fef136084ae66633e57d46/lib/router_reloader.rb#L8-L36.

Reply:
Good point, can do.